miniapps/meshing/twist.cpp

```diff
@@ -0,0 +1,279 @@
+// Copyright (c) 2010, Lawrence Livermore National Security, LLC. Produced at
+// the Lawrence Livermore National Laboratory. LLNL-CODE-443211. All Rights
+// reserved. See file COPYRIGHT for details.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability see http://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the GNU Lesser General Public License (as published by the Free
+// Software Foundation) version 2.1 dated February 1999.
+//
+//             ----------------------------------------------
+//             Twist Miniapp:  Generate simple twisted meshes
+//             ----------------------------------------------
+//
+// This miniapp generates simple periodic meshes to demonstrate MFEM's
+// implementation of periodic domains.  MFEM's strategy is to use a
+// discontinuous vector field to define the mesh coordinates on a
+// topologically periodic mesh.  It works by defining a stack of
+// individual elements and stitching together the top and bottom of
+// the mesh.  The stack can also be twisted so that the vertices of
+// the bottom and top can be joined with any integer offset (for
+// tetrahedral and wedge meshes only even offsets are supported).
+//
+// Compile with: make twist
+//
+// Sample runs:  twist
+//               twist -ns 2
+//               twist -ns -2
+//               twist -ns 2 -e 4 -pm
+//               twist -ns 2 -e 6 -pm
+//               twist -ns 3 -e 8 -pm
+//
+#include "mfem.hpp"
+#include <fstream>
+#include <iostream>
+
+using namespace std;
+using namespace mfem;
+
+static Element::Type el_type_ = Element::WEDGE;
+static int    order_ = 3;
+static int    nz_    = 3;
+static int    ns_    = 0;
```

Comment:
Isn't it better if the default for this is not `0`? Also, should the name be "number of twists"?

Reply:
Sure I can change that.